content/en/docs/tutorials/configuration/configure-java-microservice/configure-java-microservice.md

```diff
@@ -13,56 +13,181 @@ variables and then consume them using MicroProfile Config.
 
 ## {{% heading "prerequisites" %}}
 
-### Creating Kubernetes ConfigMaps & Secrets
+Before starting, make sure you have the following prerequisites:
+
+1. **Basic understanding of Kubernetes concepts**
+
+   If you're new to Kubernetes, it's essential to familiarize yourself with its core concepts. You can get started by exploring the [Kubernetes Basics](https://kubernetes.io/docs/tutorials/kubernetes-basics/) tutorial.
```

Comment:
We use site-relative links where feasible.
```suggestion
   If you're new to Kubernetes, it's essential to familiarize yourself with its core concepts. You can get started by exploring the [Kubernetes Basics](/docs/tutorials/kubernetes-basics/) tutorial.
```

Reply:
Hi @dipesh-rawat could you please guide me on how to commit to these suggestions? Should I commit directly using the "Commit Suggestion" button under your suggestion block or I have to make changes to local file and then commit again? It's my first PR and I don't know how to commit suggestion to PR.

Reply:
@gavksingh You can either use the "Commit Suggestion" button right within the suggestion block, or if you prefer, you can make the changes to the local file on your system and then commit them. Both methods work fine, so feel free to choose the one that's more convenient for you. 
Also, see https://github.com/kubernetes/website/pull/42501#discussion_r1291173661 it might be useful.

Reply:
Hi @dipesh-rawat I'm making changes to the local file which you suggested. But after making the changes to url (/docs/tutorials/kubernetes-basics/) I'm getting this error screen while trying to open the link from my device.[screenshot](https://drive.google.com/file/d/1bJLS5d50Ob_hYQgku4nuCj_-1pyawvdl/view?usp=sharing). Should I commit changes?

Reply:
> You can look how the modifications in this PR are rendered in deploy preview page at [deploy-preview-42501-configure-java-microservice/configure-java-microservice/](https://deploy-preview-42501--kubernetes-io-main-staging.netlify.app/docs/tutorials/configuration/configure-java-microservice/configure-java-microservice/) and verify whether they are as rendered as expected or not. Also, can you please add the issue number that the pull request is addressing in [description](https://github.com/kubernetes/website/pull/42501#issue-1846479882), noted as "Fixes #issuenumber"?

I mentioned the issue number. Thanks for suggesting.

Reply:
The browser takes the current domain and adds the relative path to it, effectively transforming a relative URL into an absolute one. To confirm these changes, you can try running the website on your local system (refer [README](https://github.com/kubernetes/website#readme)). This will help ensure everything is functioning as intended or you can verify from the [deploy preview](https://deploy-preview-42501--kubernetes-io-main-staging.netlify.app/docs/tutorials/configuration/configure-java-microservice/configure-java-microservice/) once you have committed the changes.

Reply:
[ScreenShot](https://drive.google.com/file/d/1rciC3GzPbRryDiZ_dszTwO-VZgJygdkj/view?usp=sharing) @dipesh-rawat could you please help me with this issue? I tried committing from my local device and somehow I got stuck at this issue? I tried [#42501(comment)](https://github.com/kubernetes/website/pull/42501#discussion_r1291173661) but somehow got stuck.

Reply:
Hi @dipesh-rawat I have made the changes that were asked. Could you please tell me what to do next?